creator-node/src/app.js

```diff
@@ -74,6 +74,9 @@ const initializeApp = (port, serviceRegistry) => {
   // add a newer version of ipfs as app property
   app.set('ipfsLatestAPI', serviceRegistry.ipfsLatest)
 
+  // Eventually, all components should pull services off the serviceRegistry
+  app.set('serviceRegistry', serviceRegistry)
```

Comment:
this is required to fix all integration tests -> by directly requiring serviceRegistry in components, the mockServiceRegistry in integration tests is not respected. by pulling serviceRegistry off app object the mock is respected